### ISSUE TITLE
After clicking on the row, I can select highlight more rows even if singleSelect property is set to true

### ISSUE THREAD

**Issue by NaomiN** [paulopmx/Flexigrid#40]
When I navigate through the grid and select a row with a click, I am able to select more rows by highlight which is confusing. Only one row should always be visible selected.

**Comment by paulopmx** [comment]
Yup. That's a wrong behavior. It could have been cost by one of the recent pull request, which I admit I haven't tested thoroughly.

If you are familiar with git, you can try using an earlier commit version. Or I can send you an older code. But the new features the other guys contributed would be missing (column state, etc.)

**Comment by NaomiN** [comment]
Sorry, I am not yet familiar with Git. I also like the grid as is - is it
possible to figure out what caused this bug and update/fix, but keep other
recent fixes?

Also, in Google Chrome and FireFox the Add/Edit/Delete buttons have images,
but in IE9 they don't.

Also, hide/unhide columns doesn't exactly work - I see the control, but I
am unable to click there. (This doesn't work in Google Chrome).

**Comment by paulopmx** [comment]
It's possible. But since this is an open-source effort and not a full-time job, you'll have to wait for a member to make time for it.

I haven't been testing for IE9. I'm waiting for some one else to make a compatibility commit.

I just tested the hide/unhide columns toggle in http://flexigrid.info/sample.html and it works on Google Chrome.

**Comment by ecaron** [comment]
Example 3 on http://flexigrid.info/, in IE9, shows the Add/Edit/Delete buttons to me.

@NaomiN, can you provide upload an attachment, or give a link, so I can reproduce the broken images you're talking about? And do so in a new issue so we don't hijack the problem you originally reported?

**Comment by NaomiN** [comment]
Eric,

I can send you the whole ASP.NET MVC 4.0 project (it is still in the early
development stage) or I can send you my version of flexigrid.js and the
view where I am using it + flexigrid.css

What do you want?

**Comment by NaomiN** [comment]
Eric,

I attached only the main files needed - the Client.cs - this is the model
file, flexigrid.js and css and images, also Client.cshtml and
ClientController.cs and two extra js files I used (Clients.js defines the
grid)

I added them individually from various folders in my project - I hope that
this may be of help. If you prefer the whole thing instead, let me know.

Thanks again.

**Comment by NaomiN** [comment]
Eric / Paulo,

Have you have a chance to look into what I sent and also into the highlight
row issue?

Thanks in advance.

**Comment by NaomiN** [comment]
Eric,

Yesterday I assigned a particular port to the application. I tested it
again right now in IE and the images do appear. Not sure why they were not
showing up before.

**Comment by mrjones66** [comment]
Souldn't line 783 read

```
if (p.singleSelect || ! g.multisel) {
```

to solve the original issue?
